package.json

```diff
@@ -103,20 +103,20 @@
     "connect-flash": "^0.1.1",
     "connect-redis": "^6.1.3",
     "csurf": "^1.11.0",
-    "dotenv": "^16.0.1",
+    "dotenv": "^16.0.2",
     "express": "^4.18.1",
     "express-prom-bundle": "^6.5.0",
     "express-session": "^1.17.3",
     "govuk-frontend": "^4.3.1",
-    "helmet": "^5.1.1",
+    "helmet": "^6.0.0",
```

Comment:
major change is insignificant for us according to change log

Reply:
yes indeed - I checked too - but good shout mentioning it on the PR